internal/validator/content.go

```diff
@@ -72,8 +72,6 @@ func (v *Validator) validateFullChunkAsync(chunk []string, allIdDiffs chan idDif
 	for _, diff := range diffs {
 		allIdDiffs <- diff
 	}
-
-	return
```

Comment:
**Rationale**:

- _Functions that have no return value do not need a return statement as the final statement of the function._